src/components/forms/ChallengeConfirmationForm.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Text, View, StyleSheet, TouchableOpacity, Image } from "react-native";
+
+import Stack from "../ui/Stack";
+import Button from "@src/components/ui/Button";
+
+const ChallengeConfirmationForm = (props: any) => {
+  const { handleCloseModalPress, handleStep } = props;
+
+  return (
+    <View style={styles.main}>
+      <Stack flexDirection="row" gap={18} p={16} justifyContent="flex-start">
+        <View style={styles.button}>
+          <TouchableOpacity
+            onPress={() => handleCloseModalPress()}
```

Comment:
nitpick: see if you could pass the handleCloseModalPress function directly, instead of wrapping it with an arrow function

```
 onPress={handleCloseModalPress} 

```

Reply:
I can! 😄 Updated the code.